go.mod

```diff
@@ -69,3 +69,5 @@ replace (
 replace github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190904142825-922d36944de3+incompatible
 
 replace github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
+
+replace github.com/integr8ly/keycloak-client => github.com/JameelB/keycloak-client v0.0.0-20200324114609-86ab7200ec70
```

Comment:
this needs to be removed after a new version has been released

Reply:
yep, i'll need to update this PR after the changes in the Keycloak client get merged.